cmd/daemon/apis/websocket.go

```diff
@@ -0,0 +1,46 @@
+package apis
+
+import (
+	"net/http"
+
+	"github.com/gorilla/websocket"
+)
+
+var upgrader = websocket.Upgrader{}
```

Comment:
Why is this stored in the global scope instead of in the scope of `HandleWebsocket`?